bundle/Resources/config/default_settings.yml

```diff
@@ -4,6 +4,7 @@ parameters:
     netgen_ez_platform_site_api.default.use_always_available_fallback: true
     netgen_ez_platform_site_api.default.fail_on_missing_fields: '%kernel.debug%'
     netgen_ez_platform_site_api.default.render_missing_field_info: false
+    netgen_ez_platform_site_api.default.named_object: []
```

Comment:
Maybe also call it with plural name to keep it in sync with config?

Reply:
Good catch, fixed in https://github.com/netgen/ezplatform-site-api/pull/145/commits/7740fefa397bbe2c0a61d1905c7ab71393db1ba4